### PR TITLE
cod1.d: whitespace formatting

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: src/dmd/backend/cod1.d [hunk 1]
```diff
@@ -124,31 +125,32 @@ struct Ssindex
...
-        {25,2,2,SSFLnobp},
-        {27,3,1,SSFLnobp},
-        {45,3,2,SSFLnobp},
-        {81,3,3,SSFLnobp},
-
-        {16,3,1,SSFLnobase1 | SSFLnobase},
-        {32,3,2,SSFLnobase1 | SSFLnobase},
-        {64,3,3,SSFLnobase1 | SSFLnobase},
+[
+    {   0, 0,0},               // [0] is a place holder
```
Comment (jacob-carlborg) [inline src/dmd/backend/cod1.d]: It's common practice to but a space after a comma.

File: src/dmd/backend/cod1.d [hunk 2]
```diff
@@ -589,134 +593,139 @@ void logexp(ref CodeBuilder cdb,elem *e,int jcond,uint fltarg,code *targ)
...
-      op != 0x8D && op != 0xC4 &&       /* and not an LEA or LES        */
-      (op != 0xFF || reg != 3) &&       /* and not CALLF MEM16          */
-      (op & 0xFFF8) != 0xD8)            // and not 8087 opcode
-  {
+    code* c,cg,cd;
+
+    debug
+    if (debugw)
+        printf("loadea: e=%p cs=%p op=x%x reg=%d offset=%lld keepmsk=%s desmsk=%s\n",
+               e,cs,op,reg,cast(ulong)offset,regm_str(keepmsk),regm_str(desmsk));
```
Comment (jacob-carlborg) [inline src/dmd/backend/cod1.d]: Space after comma.

**Comment by thewilsonator** [comment]
I stuffed up with git trying to push changes to your branch, see https://github.com/dlang/dmd/pull/8924

**Comment by thewilsonator** [comment]
Closing in favour of #8924

**Pull request closed without merging**